arangod/Aql/DistributeClientBlock.cpp

```diff
@@ -34,7 +34,7 @@ using namespace arangodb::aql;
 DistributeClientBlock::QueueEntry::QueueEntry(SkipResult const& skipped,
                                               SharedAqlItemBlockPtr block,
                                               std::vector<size_t> choosen)
-    : _skip(skipped), _block(block), _choosen(std::move(choosen)) {
+    : _skip(skipped), _block(std::move(block)), _choosen(std::move(choosen)) {
   TRI_ASSERT(_block != nullptr || choosen.empty());
```

Comment:
shouldn't this be `_choosen` in the assert? (I know this PR doesn't touch that bit, but anyway)

Reply:
it also slightly grates that this should *probably* be called `_chosen`.